.github/pull_request_template.md

```diff
@@ -1,23 +1,25 @@
-<!--
+TODO: Please describe your changes here.
```

Comment:
I think you should add a blank line after this one so the two TODO's don't end up on the same line when rendered.

Reply:
I hope that contributors mostly see this in the editor, rather than rendered, but I did add a blank line here just in case, and fiddled with a couple of other places as well.